crates/volta-core/src/tool/package_global/install.rs

```diff
@@ -0,0 +1,61 @@
+use super::Package;
+use crate::command::create_command;
+use crate::error::{Context, ErrorKind, Fallible};
+use crate::platform::PlatformSpec;
+use crate::session::Session;
+use crate::style::progress_spinner;
+use log::debug;
+
+impl Package {
+    /// Use `npm install --global` to install the package
+    ///
+    /// Sets the environment variable `npm_config_prefix` to redirect the install to the Volta
+    /// data directory, taking advantage of the standard global install behavior with a custom
+    /// location
+    pub(super) fn global_install(&self, session: &mut Session) -> Fallible<()> {
+        let package = self.to_string();
+        let default_image = session
+            .default_platform()?
+            .map(PlatformSpec::as_default)
+            .ok_or(ErrorKind::NoPlatform)?
+            .checkout(session)?;
+
+        let mut command = create_command("npm");
+        command.args(&[
+            "install",
+            "--global",
+            "--loglevel=warn",
+            "--no-update-notifier",
+            "--no-audit",
+        ]);
+        command.arg(&package);
+        command.env("PATH", default_image.path()?);
+        command.env("npm_config_prefix", self.staging.path());
+
+        debug!("Installing {} with command: {:?}", package, command);
+        let spinner = progress_spinner(&format!("Installing {}", package));
+        let output = command
+            .output()
+            .with_context(|| ErrorKind::PackageInstallFailed {
+                package: package.clone(),
+            })?;
+        spinner.finish_and_clear();
```

Comment:
This is a good question and actually a miss on my implementation: What it ultimately does is to leave it on the screen, so we should switch this up slightly to avoid that confusing output!

Reply:
This would also be resolved by #518, since we could solve it in one place and then take advantage of that everywhere.

Reply:
Updated to clear the spinner before the early return :+1:

Reply:
Ah, I'm relieved I asked this, then! Thanks for fixing it!